src/components/Layout/Header/Header.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FC, useState } from "react";
+import { Link } from "react-router-dom";
+
+import styled from "@emotion/styled";
+
+import { strings } from "../../../assets/LocalizedStrings";
+import { screenWidths } from "../../../styles/mediaBreakpoints";
+
+import "@councildataproject/cdp-design/dist/images.css";
+
+const CDPLogo = styled.div({
+  float: "left",
+  marginRight: "48px",
+});
+
+const HamburgerMenuButton = styled.button({
+  display: "block",
+  [`@media (min-width:${screenWidths.tablet})`]: {
+    // Always hidden on tablet size or above
+    display: "none",
+  },
+});
```

Comment:
This also doesn't render.

Reply:
Your screen shot looks like it is tablet size or above, which means it wouldn't render.